docker-compose.yml

```diff
@@ -27,15 +27,7 @@ services:
         expose:
             - 9000
         volumes:
-            - .:/var/www/html
-
-###########################
-# Setup the testing container
-###########################
-    testing:
-        build: ./docker/php/
-        volumes_from:
-            - php
+            - /Users/omatrot/Documents/etp_web:/etp_web
```

Comment:
I'm guessing that this worked locally. Given the configuration, I wouldn't expect it to work for anyone else, unfortunately.